docs/tutorial/part-four/index.md

```diff
@@ -270,15 +272,29 @@ export const query = graphql`
 
 `src/components/layout.js`
 
-```jsx{4,11-21,34,42-43}
+```jsx{3,25-35,38}
 import React from "react"
-import g from "glamorous"
-import { css } from "glamor"
+import { css } from "react-emotion"
 import { StaticQuery, Link } from "gatsby"
 
 import { rhythm } from "../utils/typography"
 
-const linkStyle = css({ float: `right` })
+const container = css`
```

Comment:
same here

Reply:
yeah

Reply:
I’ve forgotten this is the style Gatsby prefers. On it.